src/java_bytecode/CMakeLists.txt

```diff
@@ -18,9 +18,11 @@ add_custom_command(TARGET core_models_files
 file(GLOB_RECURSE c_sources "*.c")
 file(GLOB_RECURSE sources "*.cpp")
 file(GLOB_RECURSE headers "*.h")
+set_source_files_properties("java_core_models.inc" GENERATED)
```

Comment:
Commit can be squashed into https://github.com/cesaro/cbmc/pull/5/commits/068c730d5164434a1b3c5894d57b04e361cef962